fri/src/prover/tests.rs

```diff
@@ -16,13 +16,21 @@ type Blake3 = Blake3_256<BaseElement>;
 
 // PROVE/VERIFY TEST
 // ================================================================================================
+#[test]
+fn fri_prove_verify_exponents(){
+    let trace_length_e = 12;
+    let lde_blowup_e = 3;
+    let folding_factor_e = 2;
+    let max_remainder_size_e = 3;
+    fri_prove_verify_exponent(trace_length_e, lde_blowup_e, folding_factor_e, max_remainder_size_e)
+}
 
 #[test]
 fn fri_prove_verify() {
```

Comment:
Is there a reason not to call the new helper function from this test as well? Also, I'd prefer to revert the parameters here to what they were before (e.g., set folding factor to 4).